astropy/modeling/tests/test_pickle.py

```diff
@@ -0,0 +1,209 @@
+"""Tests that models are picklable."""
+
+from pickle import dumps, loads
```

Comment:
Huh, weird that the isort check didn't catch this. I thought it is pretty strict about separating stdlib with everything else.
```suggestion
from pickle import dumps, loads

```